### PR TITLE
Rename "dnu bundle" to "dnu publish"

### PR DESCRIPTION
**Pull request by ChengTian**
https://github.com/aspnet/dnx/issues/1474

@muratg @davidfowl

### CODE REVIEW

File: test/WebSites/LoggingWebSite/project.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "exclude": [
         "wwwroot"
     ],
-    "bundleExclude": [
+    "publishExclude": [
```
Comment (dougbu) [inline test/WebSites/LoggingWebSite/project.json]: last time we renamed project.json properties, we added warnings for the old ones but supported them for a while.  is that not happening for this change?

**Comment by davidfowl** [comment]
:shipit:

File: test/WebSites/LoggingWebSite/project.json (hunk 1 above)
Comment (davidfowl) [inline test/WebSites/LoggingWebSite/project.json]: Yes it'll happen

**Comment by muratg** [comment]
:shipit:

**Pull request merged**